### PR TITLE
Implemented basic interface for llamacheck and link to weights, adapt…

### PR DESCRIPTION
**Pull request by Ferruolo**
Provided a simple interface for llamacheck, a finetune of llama that provides spellcheck and grammarcheck functionalities. Although it runs approximately 4 seconds slower than xfspell, I believe that the added functionality is 100% worth it. Linked weights in readme

### CODE REVIEW

File: llamacheck
Comment (Galunid) [inline llamacheck:1]: You are including a binary

**Review by Galunid — CHANGES_REQUESTED** [review]
You are including a binary

File: Makefile [hunk 1]
```diff
@@ -909,6 +909,10 @@ gbnf-validator: examples/gbnf-validator/gbnf-validator.cpp ggml.o llama.o $(COMM
 	$(CXX) $(CXXFLAGS) -c $< -o $(call GET_OBJ_FILE, $<)
 	$(CXX) $(CXXFLAGS) $(filter-out %.h $<,$^) $(call GET_OBJ_FILE, $<) -o $@ $(LDFLAGS)
 
+llamacheck: examples/llamacheck/llamacheck.cpp                          ggml.o llama.o $(COMMON_DEPS) $(OBJS)
```
Comment (Galunid) [inline Makefile:912]: Please fix whitespaces

File: examples/llamacheck/llamacheck.cpp [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    if (argc == 1 || argv[1][0] == '-') {
+        printf("usage: %s MODEL_PATH [PROMPT]\n" , argv[0]);
+        return 1 ;
+    }
+
+    if (argc >= 2) {
+        params.model = argv[1];
+    }
+
+
```
Comment (Galunid) [inline examples/llamacheck/llamacheck.cpp:21]: Could you fix whitespaces in this file? There are multiple instances of double new lines and new lines after comments that shouldn't be there.

**Comment by mofosyne** [comment]
@Ferruolo is there a reason that the model card is empty in huggingface https://huggingface.co/azferruolo/llamacheck ?

**Pull request closed without merging**